src/net/libwebsocketsIO.cpp

```diff
@@ -355,6 +361,11 @@ int LibwebsocketsClient::wsCallback(struct lws *wsi, enum lws_callback_reasons r
             {
                 X509_STORE_CTX_set_error((X509_STORE_CTX*)user, X509_V_OK);
             }
+            else
+            {
```

Comment:
In case `check_public_key()` returns false, we may want to send an event to stats server in order to record the incidence of the issue.

Reply:
I'm not sure If it's possible, because wsCallback is a static method and when we retrieve LibwebsocketsClient via user pointer, it seems that it's content it's not valid.

Reply:
I see. OK, let's keep it as it is by now :)